pennylane/ops/op_math/prod.py

```diff
@@ -151,67 +151,21 @@ def circuit(weights):
         >>> qml.grad(circuit)(weights)
         array([-0.07059289])
     """
-    _name = "Prod"
-    _eigs = {}  # cache eigen vectors and values like in qml.Hermitian
-
-    def __init__(
-        self, *factors: Operator, do_queue=True, id=None
-    ):  # pylint: disable=super-init-not-called
-        """Initialize a Prod instance"""
-        self._id = id
-        self.queue_idx = None
-
-        if len(factors) < 2:
-            raise ValueError(f"Require at least two operators to multiply; got {len(factors)}")
-
-        self.factors = factors
-        self._wires = qml.wires.Wires.all_wires([f.wires for f in self.factors])
-        self._hash = None
-
-        if do_queue:
-            self.queue()
-
-    def __repr__(self):
-        """Constructor-call-like representation."""
-        return " @ ".join([f"({f})" if f.arithmetic_depth > 0 else f"{f}" for f in self.factors])
 
-    def __copy__(self):
-        cls = self.__class__
-        copied_op = cls.__new__(cls)
-        copied_op.factors = tuple(copy(f) for f in self.factors)
+    _op_symbol = "@"
 
-        for attr, value in vars(self).items():
-            if attr not in {"factors"}:
-                setattr(copied_op, attr, value)
-
-        return copied_op
+    @property
+    def factors(self):
+        """Return the factors that compose this operator."""
+        return self.operands
```

Comment:
I think we could actually just deprecate `factors` and `summands` in favour of the more general `operands`.